x-pack/platform/plugins/shared/stack_connectors/server/connector_types/openai/index.ts

```diff
@@ -44,6 +48,17 @@ export const getConnectorType = (): SubActionConnectorType<Config, Secrets> => (
   renderParameterTemplates,
 });
 
+const secretsValidator = (secretsObject: Secrets) => {
```

Comment:
Although this is schema only, the schema breaks an integration test unless we continue validating that empty secrets throw this `apiKey` error

Reply:
Ya, that's fine.  It's expected some changes like this will also need to be made with "schema-only" PRs ...